src/documents/IoCSS.md

```diff
@@ -140,3 +149,61 @@ fluid.defaults("fluid.moduleLayoutHandler", {
     }
 });
 ```
+
+## Example: `source` ##
+```javascript
+fluid.defaults("fluid.tests.uploader", {
+    gradeNames: ["fluid.component"],
+    components: {
+        uploaderContext: {
+            type: "fluid.progressiveCheckerForComponent",
+            options: {componentName: "fluid.tests.uploader"}
+        },
+        uploaderImpl: {
+            type: "fluid.tests.uploaderImpl"
+        }
+    },
+    distributeOptions: [{
+        target: "{that > uploaderImpl}.options" // Target a directly nested component matching the context "uploaderImpl"
+        source: "{that}.options",               // Distribute ALL of our options there, except exclusions:
+        exclusions: ["components.uploaderContext", "components.uploaderImpl"], // options targetted directly at these subcomponents are left undisturbed in place
+
+    }],
+    progressiveCheckerOptions: {
+        checks: [{
+            feature: "{fluid.test}",
+            contextName: "fluid.uploader.html5"
+        }]
+    }
+});
+```
+
+## Example: broadcast via root to all components ##
+
+([All Husnock, everywhere](http://en.memory-alpha.org/wiki/Husnock)) 
```

Comment:
I think this is a little bit too obscure to actually contribute anything useful to this page.